packages/gateway/README.md

```diff
@@ -39,6 +39,25 @@ You only need to `npm start` for subsequent runs. PR your env config to the `wra
 
 The IPFS gateway for nft.storage is not "another gateway", but a caching layer for NFT’s that sits on top of existing IPFS public gateways.
 
+## Usage
+
+nft.storage Gateway provides subdomain style resolutions maintaining compliance with the [same-origin policy](https://en.wikipedia.org/wiki/Same-origin_policy). The canonical form of access `https://{CID}.ipfs.nft.storage/{optional path to resource}` causes the browser to interpret each returned file as being from a different origin.
+
+```
+curl https://bafkreidyeivj7adnnac6ljvzj2e3rd5xdw3revw4da7mx2ckrstapoupoq.ipfs.nft.storage
+Hello nft.storage! 😎
+```
+
+Please note that subdomain resolution is only supported with [CIDv1](https://docs.ipfs.io/concepts/content-addressing/#identifier-formats) in case-insensitive encoding such as Base32 or Base36. For CIDv0 support, you can use the Path style resolution using the nft.storage API as follows:
+
+```
+curl https://api.nft.storage/ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX
```

Comment:
Argh, so I was hoping that `https://nft.storage/ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX` would work...is there a way?

Reply:
We should not put this on the API...

Reply:
I will see, likely I can do that! Need to play with DNS records + worker triggers, once I have https://github.com/nftstorage/nft.storage/pull/1064

I will not merge this PR until sort this out

Reply:
<img width="1030" alt="Screenshot 2022-02-08 at 15 42 08" src="https://user-images.githubusercontent.com/152863/153022352-5089de0e-ce8d-4d47-be7e-e592364e2d88.png">

Should be able to do in the same way we do this for the legacy `/api/*` route.